### PR TITLE
added default file name when formatter options not defined

### PR DESCRIPTION
**Pull request by elpikel**
https://github.com/PragTob/benchee_html/issues/16

### CODE REVIEW

File: lib/benchee/formatters/html.ex [hunk 1]
```diff
@@ -91,8 +91,17 @@ defmodule Benchee.Formatters.HTML do
...
+
+  @default_filename "benchmark_output/my.html"
+  def output(suite) do
+    suite
+    |> update_filename()
+    |> output()
+  end
+
+  defp update_filename(suite) do
+    updated_configuration = %Configuration{suite.configuration | formatter_options: %{html: %{:file => @default_filename}}}
```
Comment (devonestes) [inline lib/benchee/formatters/html.ex]: Just a small style thing here, but we use the `%{file: @default_filename}` style when we're dealing with maps that have atoms as keys.

File: test/benchee/formatters/html_integration_test.exs [hunk 2]
```diff
@@ -5,42 +5,70 @@ defmodule Benchee.Formatters.HTMLIntegrationTest do
...
+  @default_test_directory "benchmark_output"
+  @default_file_path "#{@default_test_directory}/my.html"
+  @default_comparison_path "#{@default_test_directory}/my_comparison.html"
+
   test "works just fine" do
-    basic_test time: 0.01,
-               warmup: 0.02,
-               formatters: [&Benchee.Formatters.HTML.output/1],
-               formatter_options: [html: [file: @file_path]]
+    banchee_options = [time: 0.01,
```
Comment (devonestes) [inline test/benchee/formatters/html_integration_test.exs]: Small typo here (although `banschee` would be a cool name).

Comment (devonestes) [reply test/benchee/formatters/html_integration_test.exs]: Looks like this typo is in a few places in this file. You might want to do a search and replace `s/banchee/benchee/g`

File: lib/benchee/formatters/html.ex (hunk 1 above)
Comment (elpikel) [reply lib/benchee/formatters/html.ex]: makes sense.

File: test/benchee/formatters/html_integration_test.exs (hunk 2 above)
Comment (elpikel) [reply test/benchee/formatters/html_integration_test.exs]: yep, stupid mistake - it is corrected in new commit

**Comment by elpikel** [comment]
Should be ok now

**Pull request merged**